finality-aleph/src/network/mod.rs

```diff
@@ -25,8 +25,13 @@ use component::{
 };
 use manager::SessionCommand;
 
-pub use aleph::NetworkData as AlephNetworkData;
+pub use aleph::{NetworkData as AlephNetworkData, NetworkWrapper};
+pub use component::SimpleNetwork;
+pub use manager::{ConnectionIO, ConnectionManager};
 pub use rmc::NetworkData as RmcNetworkData;
+pub use service::{Service, IO};
+pub use session::{Manager, ManagerError, Network as SessionNetwork};
```

Comment:
If you are `pub use`ing them you need to refer to this manager as `SessionManager`, otherwise it might be confused with `ConnectionManager` above.

Do you actually need these `pub use`s tho?

Reply:
You need to create `SessionManager` in `party/mod.rs`. Rename done